core-bundle/src/Resources/config/services.yml

```diff
@@ -178,6 +178,26 @@ services:
         arguments:
             - '@doctrine.orm.default_entity_manager'
 
+    contao.filesystem.dbafs_factory:
+        class: Contao\CoreBundle\Filesystem\Dbafs\DbafsFactory
+        arguments:
+            - '@database_connection'
+            - '@event_dispatcher'
+
+    contao.filesystem.dbafs_manager:
+        class: Contao\CoreBundle\Filesystem\Dbafs\DbafsManager
+
+    contao.filesystem.mount_manager:
+        class: Contao\CoreBundle\Filesystem\MountManager
+        arguments:
+            - ~
+
+    contao.filesystem.virtual_factory: # really, service linter? ;-)
```

Comment:
Maybe create a separate issue with an example if you'd expect something different here. Then this can be fixed prior to this PR (or later and we rename but the comment can be removed).

Reply:
Removed in 27cbbbe1a449528f7cb0c9e36002a209dcf06e19